sdk/cognitivelanguage/azure-ai-language-conversations/samples/sample_analyze_conversation_app.py

```diff
@@ -28,42 +30,63 @@ def sample_analyze_conversation_app():
-    conv_key = os.environ["AZURE_CONVERSATIONS_KEY"]
-    conv_project = os.environ["AZURE_CONVERSATIONS_PROJECT"]
-
-    # prepare data
-    query = "One california maki please."
-    input = ConversationAnalysisOptions(
-        query=query
-    )
+    clu_endpoint = os.environ["AZURE_CLU_ENDPOINT"]
+    clu_key = os.environ["AZURE_CLU_KEY"]
+    project_name = os.environ["AZURE_CLU_CONVERSATIONS_PROJECT_NAME"]
+    deployment_name = os.environ["AZURE_CLU_CONVERSATIONS_DEPLOYMENT_NAME"]
 
     # analyze quey
-    client = ConversationAnalysisClient(conv_endpoint, AzureKeyCredential(conv_key))
+    client = ConversationAnalysisClient(clu_endpoint, AzureKeyCredential(clu_key))
     with client:
-        result = client.analyze_conversations(
-            input,
-            project_name=conv_project,
-            deployment_name='production'
+        query = "Send an email to Carol about the tomorrow's demo"
+        result = client.analyze_conversation(
+            task={
+                "kind": "CustomConversation",
+                "analysisInput": {
+                    "conversationItem": {
+                        "participantId": "1",
+                        "id": "1",
+                        "modality": "text",
+                        "language": "en",
+                        "text": query
+                    },
+                    "isLoggingEnabled": False
+                },
+                "parameters": {
+                    "projectName": project_name,
+                    "deploymentName": deployment_name,
+                    "verbose": True
+                }
+            }
         )
 
     # view result
-    print("query: {}".format(result.query))
-    print("project kind: {}\n".format(result.prediction.project_kind))
+    print("query: {}".format(result.results.query))
+    print("project kind: {}\n".format(result.results.prediction.project_kind))
+
+    print("top intent: {}".format(result.results.prediction.top_intent))
+    print("category: {}".format(result.results.prediction.intents[0].category))
+    print("confidence score: {}\n".format(result.results.prediction.intents[0].confidence))
 
-    print("view top intent:")
-    print("\ttop intent: {}".format(result.prediction.top_intent))
-    print("\tcategory: {}".format(result.prediction.intents[0].category))
-    print("\tconfidence score: {}\n".format(result.prediction.intents[0].confidence_score))
+    print("entities:")
+    for entity in result.results.prediction.entities:
+        print("\ncategory: {}".format(entity.category))
+        print("text: {}".format(entity.text))
+        print("confidence score: {}".format(entity.confidence))
+        if entity.resolutions:
+            print("resolutions")
+            for resolution in entity.resolutions:
+                print("kind: {}".format(resolution.resolution_kind))
+                print("value: {}".format(resolution.additional_properties["value"]))
```

Comment:
This looks like an error in the swagger spec - these should be deserializing to objects with a value attribute right?

Reply:
Let's open an issue for this and address it separately.